app/components/task_list_items/assessment/considerations_component.rb

```diff
@@ -3,8 +3,10 @@
 module TaskListItems
   module Assessment
     class ConsiderationsComponent < TaskListItems::BaseComponent
-      def initialize(planning_application:)
+      def initialize(planning_application:, link_text: nil, link_path: nil)
```

Comment:
I'm allow the component to override the link and text so we can reuse the existing component - is the intention to replace the original considerations design with the new one?